app/src/components/Header.tsx

```diff
@@ -1,29 +1,16 @@
 import React from 'react';
-import { Platform, StyleSheet, Text, View } from 'react-native';
-
-const styles = StyleSheet.create({
-  android: {
-    top: 35,
-  }
-});
+import { Text, View } from 'react-native';
 
 const Header = () => {
-  if (Platform.OS == 'android')
-    return (
-      <View style={styles.android} className="items-center justify-between flex-row px-5">
-        <Text className="text-white">Settings</Text>
-        <Text className="font-bold text-xl text-white">Umbrella?</Text>
-        <Text className="text-white">Notifications</Text>
-      </View>
-    );
-  else
-    return (
-      <View className="items-center justify-between flex-row px-5">
-        <Text className="text-white">Settings</Text>
-        <Text className="font-bold text-xl text-white">Umbrella?</Text>
-        <Text className="text-white">Notifications</Text>
-      </View>
-    );
+  return (
+    <View className="items-center justify-between flex-row px-5 android:mt-4">
+      <Text className="text-white font-zenMaru">Settings</Text>
+      <Text className="font-bold text-xl text-white font-zenMaru">
+        Umbrella?
```

Comment:
Logo is in lowercase format with "black" weight, like suggested on Figma = "umbrella?"